style.css

```diff
@@ -1,3 +1,18 @@
+@media (prefers-color-scheme: dark) {
+    body {
+        background: #1e1e1e;
+        color: #cfcfcf;
+    } 
+    input, thead {
+        background: #e9e9edde !important;          
+        color: #222 !important;
```

Comment:
Please stick to six digit format, and avoid `!important` unless there's a strong reason in favor.